### DISCUSSION TITLE
Map audio devices to screens

### DISCUSSION THREAD

**Discussion by Jadro007** [File-New-Project/EarTrumpet#640] [Ideas]
Hello :)
are you multitasking? Watching video and gaming at the same time? Or just listening youtube on you external monitor while working? And are using different audio devices for that? In that case keep reading!

I really love this projects and I have an idea, which could be useful (at least for me). Lets see what others think. 

After some time using multiple screens and multiple devices (in my case laptop with internal speakers and external monitor with speakers behind it) I found out that I always switch apps sound on specific screens to specific speakers. So would'd it be useful to automatically switch audio device for app when you move it to different screen? 

If you read it until here, feel free to comment what you think about it and how would you improve it. Or just tell me why it is bad idea.. ;)

I plan to implement it (at least for my personal use) and if you like it, I can push it here :)

**Comment by pixelatedmirror** [comment]
I was just going to add a feature request for this, so consider this a vote. I'd love to have that functionality.
I have 2 machines, 2 KVMs, and 2 monitors with built-in speakers. I'll often be using both monitors on one computer, then switch one of them to the other computer. If I have something running, I'll drag it to the unchanged monitor, but if I'm switching the main monitor, I lose audio unless I also switch output devices before switching computers.
It would also be useful for sending media (streaming, online) from the computer to a TV connected to the same PC.
